### ISSUE TITLE
Export HelpWindow to the Workbench

### ISSUE THREAD

**Issue by martyngigg** [mantidproject/mantid#24709]
The [`HelpWindow`](https://github.com/mantidproject/mantid/blob/master/qt/widgets/common/inc/MantidQtWidgets/Common/HelpWindow.h) class is used to provide the offline help for MantidPlot. The same class should be able to be used for the workbench Python code. It needs to be accessible from Python so will require exporting and having wrapper modules provided. Once exposed to Python the current python references that access the `HelpWindow` should be updated to use `mantidqt`.

The `HelpWindow` needs to be included in the Qt5 build of the `MantidWidgetsCommon` library and when this is done the `?` button on the algorithm dialogs should work. This needs to be verified.

**Comment by peterfpeterson** [comment]
Would rewriting this in python be better? https://doc.qt.io/qtforpython/overviews/qthelp-framework.html

**Comment by martyngigg** [comment]
It's an idea. Just watch out with that link as it's about the Qt projects new Python bindings and not `PyQt`. `PyQt5` does have QtHelp bindings though. @hsautessella can probably comment on this further.

**Comment by peterfpeterson** [comment]
My recommendation is to try bringing up the help system in python against our existing `qhc` file and see what happens.

**Comment by hsautessella** [comment]
Sorry for the delayed response, I had to remind myself exactly where I was up to before the release period. 

I have got the help menu running locally in Workbench via the `InterfaceManager`. The main issue I was having was (on Windows) Qt was looking in the wrong place for the executable and resource files that QtWebEngine needs ([detailed here](https://doc.qt.io/qt-5/qtwebengine-deploying.html#deploying-qt-webengine-processes)).

This is solvable by creating a `qt.conf` file in `external\src\ThirdParty\lib\python2.7\` that points Qt to the correct place So the `qt.conf` file simply contains: 
```
[Paths]
Prefix = ../qt5
```
I'm not familiar with the packaging process so I don't know if this raises some packaging issues or not. Or whether or not rewriting in Python would solve this.
